architecture/topics/aws_getting_started_ansible.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * admin_guide/aws-getting-started.adoc
+
+[id='aws-getting-started-ansible-{context}']
+= Binding applications to Ansible playbooks
```

Comment:
I don't follow why this is referencing 'Ansible playbooks', my understanding is the new AWS Service Broker no longer uses APBs nor Ansible.